daemon/container_windows.go

```diff
@@ -18,6 +18,8 @@ import (
 // the container. Docker has no context of what the default path should be.
```

Comment:
I think it only triggers on functions/methods and structs. I will format this comment so it matches.